applications/asset_tracker_v2/src/modules/util_module.c

```diff
@@ -211,16 +211,12 @@ static void send_reboot_request(void)
  * modules in the application. When this API has been called a set number of times equal to the
  * number of active modules, a reboot will be scheduled.
  */
-static void reboot_ack_check(void)
+static void reboot_ack_check(uint32_t module_id)
 {
-	static uint8_t reboot_ack_count;
-
-	reboot_ack_count++;
-
 	/* Reboot after a shorter timeout if all modules have acknowledged that they are ready
 	 * to reboot, ensuring a graceful shutdown.
 	 */
-	if (reboot_ack_count >= module_active_count_get() - 1) {
+	if (modules_shutdown_register(module_id)) {
```

Comment:
What happens if not all modules report a shutdown?

Reply:
Added a comment explain what occurs if not all modules acknowledge the shutdown request.